### PR TITLE
.Net: Add CancellationToken parameters to to a few methods

### PR DESCRIPTION
**Pull request by SergeyMenshykh**

### CODE REVIEW

File: dotnet/src/SemanticKernel.Core/KernelExtensions.cs [hunk 1]
```diff
@@ -514,12 +514,14 @@ public static IKernelBuilderPlugins AddFromObject(this IKernelBuilderPlugins plu
     /// <exception cref="ArgumentException"><paramref name="promptTemplate"/> is empty or composed entirely of whitespace.</exception>
     /// <exception cref="KernelFunction">The function failed to invoke successfully.</exception>
     /// <exception cref="KernelFunctionCanceledException">The <see cref="KernelFunction"/>'s invocation was canceled.</exception>
+    /// <param name="cancellationToken">The cancellation token.</param>
```
Comment (stephentoub) [inline dotnet/src/SemanticKernel.Core/KernelExtensions.cs:517]: Can you use the same comment as on
https://github.com/microsoft/semantic-kernel/pull/4298/files#diff-1966179877500641f0ce2da0026cb59eecdd711eccab3d58e53b0aaca6ba05efL546

File: dotnet/src/Extensions/PromptTemplates.Handlebars/Extensions/HandlebarsKernelExtensions.cs [hunk 2]
```diff
@@ -18,13 +19,15 @@ public static class HandlebarsKernelExtensions
     /// <param name="kernel">The <see cref="Kernel"/> containing services, plugins, and other state for use throughout the operation.</param>
     /// <param name="promptTemplate">Plain language definition of the prompt, using Handlebars prompt template language</param>
     /// <param name="arguments">The operation arguments</param>
+    /// <param name="cancellationToken">The cancellation token.</param>
```
Comment (stephentoub) [inline dotnet/src/Extensions/PromptTemplates.Handlebars/Extensions/HandlebarsKernelExtensions.cs:22]: For all of these, can you use the same comment that's used on the other token params?

Comment (stephentoub) [inline dotnet/src/Extensions/PromptTemplates.Handlebars/Extensions/HandlebarsKernelExtensions.cs:22]:
```suggestion
    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
```

File: dotnet/src/SemanticKernel.Core/KernelExtensions.cs (hunk 1 above)
Comment (stephentoub) [inline dotnet/src/SemanticKernel.Core/KernelExtensions.cs:517]:
```suggestion
    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
```

**Review by stephentoub — APPROVED** [review]

**Review by dmytrostruk — APPROVED** [review]

**Review by markwallace-microsoft — APPROVED** [review]

**Pull request merged**